### PR TITLE
Первоначальная страница

### PR DESCRIPTION
**Pull request by NikitaGoroshko7**
Первоначальная страница
https://trello.com/c/x8qZ3Wzn/2-%D0%BF%D0%B5%D1%80%D0%B2%D0%BE%D0%BD%D0%B0%D1%87%D0%B0%D0%BB%D1%8C%D0%BD%D0%B0%D1%8F-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0
Added action and view for welcome page.
Added style and script for view.
Added images for view.

### CODE REVIEW

File: CrowdFundingApplication/Controllers/HomeController.cs [hunk 1]
```diff
@@ -28,6 +28,8 @@ public IActionResult Privacy()
             return View();
         }
 
+        public IActionResult HomePage() => View();
```
Comment (GalayM) [inline CrowdFundingApplication/Controllers/HomeController.cs]: Please use Index action name as default page

File: CrowdFundingApplication/wwwroot/css/HomePage.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
+﻿/* fonts */
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/css/HomePage.css]: Where are these styles from?

File: CrowdFundingApplication/wwwroot/js/HomePageScript.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+﻿//////////////////////////////////////////////////////////
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/js/HomePageScript.js]: Why this functionality? The home page should be as simple as possible.

File: CrowdFundingApplication/wwwroot/js/HomePageScript.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+//появление navbar'a
+$(window).scroll(function () {
+    if ($(window).scrollTop() > 0) {
+        $('.navbar').css('background', '#212529');
+    } else {
+        $('.navbar').css('background', 'transparent');
+    }
+})
+//////////////////////////////////////////////////////////
+//авто скролл до определенной секции
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/js/HomePageScript.js]: All comments in English

File: CrowdFundingApplication/wwwroot/css/HomePage.css (hunk 2 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/wwwroot/css/HomePage.css]: I took this style from the Internet

**Pull request merged**